recipes/grpc/all/conandata.yml

```diff
@@ -1,24 +1,33 @@
 sources:
-  "1.50.1":
-    url: "https://github.com/grpc/grpc/archive/v1.50.1.tar.gz"
-    sha256: "fb1ed98eb3555877d55eb2b948caca44bc8601c6704896594de81558639709ef"
-  "1.50.0":
-    url: "https://github.com/grpc/grpc/archive/v1.50.0.tar.gz"
-    sha256: "76900ab068da86378395a8e125b5cc43dfae671e09ff6462ddfef18676e2165a"
-  "1.47.1":
-    url: "https://github.com/grpc/grpc/archive/v1.47.1.tar.gz"
-    sha256: "ba7c36989ad8a52c813adeebb0c65869283448f496dbb2868c75f69941dec6a9"
-  "1.46.3":
-    url: "https://github.com/grpc/grpc/archive/refs/tags/v1.46.3.tar.gz"
-    sha256: "d6cbf22cb5007af71b61c6be316a79397469c58c82a942552a62e708bce60964"
+  "1.54.2":
+    url: "https://github.com/grpc/grpc/archive/v1.54.2.tar.gz"
+    sha256: "3c305f0ca5f98919bc104448f59177e7b936acd5c69c144bf4a548cad723e1e4"
+  "1.50.2":
+    url: "https://github.com/grpc/grpc/archive/v1.50.2.tar.gz"
+    sha256: "e941752638a92f21e27566903e1eb9b4a6380d70ca5ce91116b4cff6cb8fe2f2"
+  "1.48.4":
+    url: "https://github.com/grpc/grpc/archive/v1.48.4.tar.gz"
+    sha256: "0c3faa83e39d4f1ab55fe1476362b9ac3b81632a46dce7fd4d50271bce816b53"
+  "1.47.5":
+    url: "https://github.com/grpc/grpc/archive/v1.47.5.tar.gz"
+    sha256: "bf47a9b6903f0b6ca77c4584601ea6ef93efdad905e20b78a23417a362bc2190"
+  "1.46.7":
+    url: "https://github.com/grpc/grpc/archive/v1.46.7.tar.gz"
+    sha256: "1f9cf306a79e9a76ff36f2f0563b72c84ab9f2592372a742a234f360ae733e54"
```

Comment:
```suggestion
```
I would suggest by removing older versions, as this project is big and requires a good amount of CI time. It's allowed to remove old versions: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-is-the-policy-for-removing-older-versions-of-a-package

Reply:
👍 make sense, keeping 1.48 as latest in 1.4x.x branch, it has some behavior changes between 1.4x.x and 1.5x.x